CHANGELOG.md

```diff
@@ -10,8 +10,9 @@ and this project adheres to
 
 - cosmwasm-std: Implement `Decimal{,256}::checked_mul` and
   `Decimal{,256}::checked_pow`.
-- cosmwasm-std: Implement `Sub`/`SubAssign` for `Uint64`.
-- cosmwasm-std: Implement `Mul`/`MulAssign` for `Uint64`.
+- cosmwasm-std: Implement `Sub`/`SubAssign` for `Uint64`/`Uint128`/`Uint256`/`Uint512`.
```

Comment:
I just noticed that @webmaster128 put only `Uint64` in changelog, while implementing for all other types as well.
Intentional or by mistake?

Reply:
It was implemented for the oother types already. So the change was only for Uint64.